hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/operation/SnapshotCompleteOperation.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.impl.operation;
+
+import com.hazelcast.jet.impl.JetService;
+import com.hazelcast.jet.impl.execution.ExecutionContext;
+import com.hazelcast.jet.impl.execution.init.JetInitDataSerializerHook;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+
+import static com.hazelcast.jet.impl.util.LoggingUtil.logFine;
+
+public class SnapshotCompleteOperation extends AsyncJobOperation {
```

Comment:
would it better to call this SnapshotPhase1CompleteOperation? or at least document it as such.